row_store.go

```diff
@@ -111,6 +111,8 @@ func (t *table) openRowStore(opts *rowStoreOptions) (*rowStore, wal.Offset, erro
 				} else {
 					walOffset = wal.Offset(o)
 				}
+				// clear existingFileName so we don't use it for filestore
+				existingFileName = ""
```

Comment:
Drive-by fix. Previously, for tables with no data, we set the offset file as the filename. When trying to iterate, this would cause an error since the offset file is not a valid data file.